server/templates/static/homepage.html

```diff
@@ -57,119 +57,207 @@ <h2>{% trans %}What's new{% endtrans %}</h2>
   </div>
 </section>
 
-<section class="container">
+<section class="container examples">
   {# TRANSLATORS: Header for a list of links to Data Commons data displays. Please see this string in action at https://datacommons.org #}
   <h2>{% trans %}Explore the data{% endtrans %}</h2>
   <p>
-    {# TRANSLATORS: Describes how we clean and process data for import once, usually a painful part of data analysis. Data is imported into a unified graph (e.g. data about California from many sources are all added to the same node in our graph. Please see this string in action at https://datacommons.org #}
-    {% trans -%}
-    HOME-EXPLORE_CALLOUT
-    {%- endtrans %}
+    We aggregated data across a wide range of topics for easy exploration.
   </p>
-  <ul>
+
+  <ul class="mt-4">
+    <li>
+      <span class="main">IPCC Temperature projections</span>:
+      <ul>
+        <li>Max Temperatures under RCP 4.5 in
+          <a class="sub" href="{{ url_for('tools.map') }}#sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&pd=country/IND&pn=India&pt=Country&ept=AdministrativeArea2">{{ place_names['country/IND'] }}</a> and
+          <a class="sub" href="{{ url_for('tools.map') }}#sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&pd=country/USA&pn=United%20States%20of%20America&pt=Country&ept=County">{{ place_names['country/USA'] }}</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&svy=Percent_Person_WithCoronaryHeartDisease&epd=country/USA&epn=United%20States%20of%20America&epts=Country&ept=County&qd=1&ld=0&dd=1&ct=0&rg=0">Prevalence Of Coronary Heart Disease vs Max Temperature under RCP 4.5 for USA Counties</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.timeline') }}#statsVar=DifferenceAcrossModels_Max_Temperature&place=geoId/0668000">Difference across models for {{ place_names['geoId/0668000'] }}</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=NumberOfMonths_WetBulbTemperature_35COrMore_RCP45_MinRelativeHumidity&pc=0&denom=Count_Person&dt=2099&pd=country/IND&pn=India&pt=Country&ept=AdministrativeArea2">Districts in {{ place_names['country/IND'] }} with Wet Bulb Temp > 35°C</a>
```

Comment:
nit: "with Wet-bulb temperature over 35'c" ...

(Wonder if its worth linking to https://en.wikipedia.org/wiki/Wet-bulb_temperature ?)